docs/usage.md

```diff
@@ -956,6 +956,8 @@ Since both Snpeff and VEP are internally figuring the path towards the specific
 
 So if you are using this resource, please either use the `--use_annotation_cache_keys`, or point towards the specific species, genome and build matches the directory structure within the cache.
```

Comment:
I am stillam still mildly fussy on this, but from what i understand now: `--use_annotation_cache_keys` is the equivalent to `igenomes_base`. Would i make sense to give it a similar name: `--annotation_cache_base` and set it b default to the AWS ressource, same as igenomes?

Reply:
not really actually.
`vep_cache` and `snpeff_cache` would be equivalent to `igenomes_base`.
`use_annotation_cache_keys` is just a boolean so that Sarek knows there is an extra layer in the organisation of the cache because of cloud related logic.

Reply:
```suggestion
So if you are using this resource, please either set `--use_annotation_cache` to use the AWS annotation cache, or point towards the specific species, genome and build matches the directory structure within your own cache folder structure.
```